_includes/pcmdi_nav.html

```diff
@@ -63,11 +63,13 @@
                         </a>
                         <ul class="dropdown-menu">
                             <li><a href="{{site.baseurl}}/CMIP6/">Overview</a></li>
-                            <li><a href="{{site.baseurl}}/CMIP6/Guide/dataUsers.html">Data Users Guide</a></li>
-                            <li><a href="{{site.baseurl}}/CMIP6/ArchiveStatistics/esgf_data_holdings/">Current Data Holdings</a></li>
-                            <li><a href="http://esgf-node.llnl.gov/search/cmip6">CMIP6 Data Search</a></li>
+                            <li><a href="{{site.baseurl}}/CMIP6/Guide/modelers.html">Guide for data users</a></li>
+                            <li><a href="{{site.baseurl}}/CMIP6/Guide/dataUsers.html">Guide for participating modelers</a></li>
```

Comment:
The users and modelers links got switched.

Reply:
@mauzey1 great catch, I will fix and re-push